ck.sh

```diff
@@ -149,7 +153,7 @@ case "$COMMAND" in
 			echo
 			echo "This will reset the \"patches\" directory and replace these patches:"
 			echo
-			ls ../patches/*.patch | cat
+			find ../patches/*.patch | cat
```

Comment:
shellcheck again?

The thing about this is you still have glob expansion going on, so both versions of this command get expanded as though you had typed (wrapped for readability):

```
ls ../patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch \
  ../patches/0002-LPS-95472-Tabs-in-popups-not-appears-correctly-in-ma.patch \
  ../patches/0003-LPS-85326-Remove-check-for-Webkit-browsers.patch \
  ../patches/0004-LPP-36989-Remove-obsolete-summary-field-from-table-e.patch | cat
```

or:

```
find ../patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch \
  ../patches/0002-LPS-95472-Tabs-in-popups-not-appears-correctly-in-ma.patch \
  ../patches/0003-LPS-85326-Remove-check-for-Webkit-browsers.patch \
  ../patches/0004-LPP-36989-Remove-obsolete-summary-field-from-table-e.patch | cat
```

Now, I imagine that the intent of the shellcheck advice is actually to avoid shell expansion, not to avoid `ls` as such... so if you want to do that, the command would be:

```
find ../patches -name '*.patch'
```

- No glob expansion, because of the single quotes.
- No useless `cat` (`find`, unlike `ls`, is always going to print one result per line, so it never needed `cat`; the purpose of it in `ls | cat` is to force one item per line).

Reply:
shellcheck, yep